packages/CodingStyle/src/Rector/FuncCall/FunctionCallToConstantRector.php

```diff
@@ -0,0 +1,191 @@
+<?php declare(strict_types=1);
+
+namespace Rector\CodingStyle\Rector\FuncCall;
+
+use PhpParser\Node;
+use PhpParser\Node\Expr;
+use PhpParser\Node\Expr\BinaryOp\Greater;
+use PhpParser\Node\Expr\BinaryOp\GreaterOrEqual;
+use PhpParser\Node\Expr\BinaryOp\Identical;
+use PhpParser\Node\Expr\BinaryOp\NotIdentical;
+use PhpParser\Node\Expr\BinaryOp\Smaller;
+use PhpParser\Node\Expr\BinaryOp\SmallerOrEqual;
+use PhpParser\Node\Expr\ConstFetch;
+use PhpParser\Node\Expr\FuncCall;
+use PhpParser\Node\Name;
+use PhpParser\Node\Scalar\LNumber;
+use PhpParser\Node\Scalar\String_;
+use Rector\Exception\ShouldNotHappenException;
+use Rector\Rector\AbstractRector;
+use Rector\RectorDefinition\CodeSample;
+use Rector\RectorDefinition\RectorDefinition;
+
+/**
+ * @see \Rector\CodingStyle\Tests\Rector\FuncCall\FunctionCallToConstantRector\FunctionCallToConstantRectorTest
+ */
+final class FunctionCallToConstantRector extends AbstractRector
```

Comment:
When I read `FunctionCallToConstantRector`, I thought it's something like:

```diff
-get_value()
+GET_VALUE;
```

From the tests I'd assume there are 2 rules:

- `VersionCompareFuncCallToPHPVersionConstantRector`

and configurable generic rule `FunctionCallToConstantRector`:

```yaml
FunctionCallToConstantRector:
    $functionsToConstants:
        php_sapi_name: PHP_SAPI
        pi: M_PI
```